tests/Calculator/CalculatorTest.php

```diff
@@ -84,6 +84,10 @@ public function testRound()
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_EVEN));
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_ODD));
         $this->assertEquals('12', $calculator->round('12.50', Money::ROUND_HALF_DOWN));
+        $this->assertEquals('-2', $calculator->round(-1.5, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.578947368, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.5, Money::ROUND_HALF_UP));
```

Comment:
@pamil While I totally agree that it is misleading, I am not sure that this package should move away from that (misleading) behaviour. The PHP behaviour is the one we had in version 1 of this package. It is very well explainable that we follow the PHP behaviour by default. We could introduce new rounding modes: `Money::ROUND_HALF_TOWARDS_POSITIVE_INFINITY` and `Money::ROUND_HALF_TOWARDS_NEGATIVE_INFINITY`. We could even change the default behaviour.

@sagikazarmark What do you think?